src/main/java/core/basesyntax/storage/FruitStorage.java

```diff
@@ -0,0 +1,25 @@
+package core.basesyntax.storage;
+
+import core.basesyntax.model.Operation;
+import java.util.List;
+import java.util.Map;
+
+public class FruitStorage {
+    private static final int OPERATION_TYPE_INDEX = 0;
+    private static final String SEPARATOR = ",";
+    private final ProcessingData processingData;
+
+    public FruitStorage(ProcessingData processingData) {
+        this.processingData = processingData;
+    }
+
+    public Map<Operation, Map<String, List<Integer>>> createStorage(Map<Operation, Map<String,
+            List<Integer>>> fruitMaps, List<String> listOperation) {
+        for (String rowFromOperationList : listOperation) {
+            String[] split = rowFromOperationList.split(SEPARATOR);
+            String operationType = split[OPERATION_TYPE_INDEX];
+            processingData.recognitionData(fruitMaps,operationType,rowFromOperationList);
+        }
+        return fruitMaps;
+    }
+}
```

Comment:
This class should just emulate DB where intermediate result of processing operations will be stored
Here we should have map of fruitName/quantity
```suggestion
public class FruitStorage {
    public static final Map<String, Integer> storage;
}
```